### ISSUE TITLE
Tell users if they enter an invalid age

### ISSUE THREAD

**Issue by nworden** [google/personfinder#592]
PFIF requires that age values either be a number or arrange of numbers (formatted as nn-nn). We currently enforce that requirement silently: if you enter something invalid, the form submission will go through, and we'll just drop the age value. We should tell users if the age they submitted is invalid and give them a chance to fix it.

**Comment by gimite** [comment]
It may also be nice if we are a bit more flexible about the accepted format e.g., accepting full with numbers (e.g., "３０−３５") as well, which is commonly used in CJK. Applying Unicode NFC normalization would probably be sufficient but I haven't confirmed it.

**Comment by yashLadha** [comment]
Would like to take up the issue @nworden @gimite

**Comment by gabru-md** [comment]
if this is a good first issue then can I take it up @yashLadha ?

**Comment by yashLadha** [comment]
No problem in that @gabru-md :sweat_smile:

**Comment by pollyMath** [comment]
If this is not worked on and it is still relevant, I would like to take it :)

**Comment by routrohan** [comment]
Is this issue still open?
If so, I would like to work upon it.

**Comment by nworden** [comment]
Yeah, there hasn't been follow-up on the above PR, so feel free to take it. Thanks!

**Comment by pollyMath** [comment]
sorry for great delay, have done another commit to my PR

**Comment by darshang99** [comment]
I'm a beginner to this project can I work on this issue and point me to the resources. Is this still open? I would like to work on it.
